meta-openpower/recipes-phosphor/host/op-proc-control_git.bb

```diff
@@ -2,6 +2,7 @@ SUMMARY = "OpenPower procedure control"
 DESCRIPTION = "Provides procedures that run against the host chipset"
 PR = "r1"
 PV = "1.0+git${SRCPV}"
+HOMEPAGE = "https://github.com/openbmc/openpower-proc-control"
```

Comment:
This seems un-necessary?

Reply:
Hi Andrew, as per latest discussion @rfrandse is going to bump up rev